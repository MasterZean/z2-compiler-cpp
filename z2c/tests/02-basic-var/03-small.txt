val a = -0o0s8
val b = 0o0s8
val c = -0o1s8
val d = 0o1s8
val e = -0o144s8
val f = 0o144s8
val g = -0o177s8
val h = 0o177s8
val i = -0o200s8
val j = 0o200s8
val k = -0o310s8
val l = 0o310s8