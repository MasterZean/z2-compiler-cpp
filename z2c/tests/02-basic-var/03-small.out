int8 a = 0000;
int8 b = 0000;
int8 c = -0001;
int8 d = 0001;
int8 e = -0144;
int8 f = 0144;
int8 g = -0177;
int8 h = 0177;
int8 i = -0200;
DummyClass(10, 9): error:
	'Small' literal constant too big
DummyClass(11, 9): error:
	'Small' literal constant too big
DummyClass(12, 9): error:
	'Small' literal constant too big
